.typos.toml

```diff
@@ -44,3 +44,4 @@ PyCodeInterupt = "PyCodeInterupt"
 simin_nam = "simin_nam"
 simout_nam = "simout_nam"
 displayAvailablityWarning = "displayAvailablityWarning"
+indx = "indx"
```

Comment:
Where is this being used?

Reply:
There's a bunch of places where a var `indx` is used and typos now wants to change that to `index`.

Take a look at [JB's PR test failures](https://github.com/CCSI-Toolset/FOQUS/actions/runs/20082742668/job/57613458398?pr=1274) for what motivated this PR.

The alternative is to just change all those var to be "index" and make the above exception, I could go either way on that, I just chose the this out of laziness.